index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
```

Comment:
I should have used main tag instead of creating a div and adding id as main . Using the main tag follows the html 
semantics and it makes the code look more organised